ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -0,0 +1,422 @@
+                                            <strong>Subscription Viewer</strong> role. See{' '}
+                                            <Button
+                                                variant="link"
+                                                isInline
+                                                component="a"
+                                                href="https://docs.redhat.com/en/documentation/openshift_cluster_manager/1-latest/html-single/managing_clusters/index#assembly-user-management-ocm"
+                                                target="_blank"
+                                                rel="noreferrer"
+                                            >
+                                                Configuring access to clusters in OpenShift Cluster
+                                                Manager
+                                            </Button>{' '}
+                                            for more information.
+                                        </FlexItem>
+                                        <FlexItem>
+                                            <em>
+                                                Service accounts are the preferred authentication
+                                                method over the deprecated API token.
+                                            </em>
+                                        </FlexItem>
+                                    </Flex>
+                                }
+                                ariaLabel="Help for client secret"
+                            />
+                        }
+                        fieldId="cloudSource.credentials.clientSecret"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired={
+                                values.updateCredentials && !values.cloudSource.credentials.secret
+                            }
+                            type="password"
+                            id={`cloudSource.credentials.clientSecret`}
+                            value={values.cloudSource.credentials.clientSecret}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable || !values.updateCredentials}
+                            placeholder={
+                                values.updateCredentials
+                                    ? ''
+                                    : 'Currently-stored client secret will be used.'
+                            }
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        isRequired={values.updateCredentials}
+                        label="API token (deprecated)"
```

Comment:
Ditto the previous comments:
![API_token](https://github.com/user-attachments/assets/1907f862-15c6-4937-a492-1b793a52bbea)

```tsx
                        label="API token (deprecated)"
                        labelIcon={
                            <IntegrationHelpIcon
                                hasAutoWidth
                                helpTitle="API token"
                                helpText={
                                    <Flex direction={{ default: 'column' }}>
                                        <FlexItem>
                                            <ExternalLink>
                                                <a
                                                    href="https://console.redhat.com/openshift/token"
                                                    target="_blank"
                                                    rel="noopener noreferrer"
                                                >
                                                    OpenShift Cluster Manager offline token
                                                </a>
                                            </ExternalLink>
                                        </FlexItem>
                                        <FlexItem>
                                            <em>
                                                Service accounts are the preferred authentication
                                                method over the deprecated API token.
                                            </em>
                                        </FlexItem>
                                    </Flex>
                                }
                                ariaLabel="Help for API token"
                            />
                        }
```

Reply:
done, see [2754468](https://github.com/stackrox/stackrox/pull/12277/commits/27544685b601615d3fd113fbc08622e40c1599e3)